Cargo.toml

```diff
@@ -105,9 +109,18 @@ branch = "dev"
 features = ["profiling"]
 optional = true
 
+[dev-dependencies]
+test_util = { path = "components/test_util" }
+test_raftstore = { path = "components/test_raftstore" }
+test_storage = { path = "components/test_storage" }
+criterion = "0.2"
+
 [workspace]
 members = [
-  "fuzz"
+  "fuzz",
+  "components/test_raftstore",
+  "components/test_storage",
+  "components/test_util"
```

Comment:
What about just one component contains `test_raftstore`, `test_storage` and `test_util`?